pkg/api/v1/client/direct_transport.go

```diff
@@ -0,0 +1,203 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/kubeshop/testkube/pkg/executor/output"
+	phttp "github.com/kubeshop/testkube/pkg/http"
+	"github.com/kubeshop/testkube/pkg/problem"
+	"golang.org/x/oauth2"
+)
+
+// NewDirectTransport returns new proxy transport
```

Comment:
not sure if `Transport` is good name here - I feel transport more like  HTTP / GRPC thing this one is more like `Client`?

Reply:
renamed